code/src/App.js

```diff
@@ -1,12 +1,26 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './components/Album'
+import { Header } from './components/Header'
+import './App.css'
 
 console.log(data)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="albumComponent">
+      <div className="headerSection">
```

Comment:
No need to wrap `<Header /> ` in a div, you can put a className on that instead.